src/main/java/uk/gov/pay/adminusers/persistence/entity/service/ServiceNameEntity.java

```diff
@@ -42,6 +42,14 @@ public ServiceNameEntity() {
         // make JPA happy
     }
 
+    public static ServiceNameEntity from(SupportedLanguage language, String name) {
```

Comment:
As discussed you can use `SupportedLanguage` from our shared library in the next pr